src/post.ts

```diff
@@ -1,12 +1,12 @@
-import { Mode, Renderer } from './embed';
+import { Config, Mode, Renderer } from './embed';
 
 /**
  * Open editor url in a new window, and pass a message.
  */
 export function post(
   window: Window,
   url: string,
-  data: { config: string; mode: Mode; renderer: Renderer; spec: string }
+  data: { config: Config; mode: Mode; renderer: Renderer; spec: string }
```

Comment:
This cannot be string so the type could be stronger.

Reply:
Done!